pypeit/spectrographs/p200_dbsp.py

```diff
@@ -0,0 +1,432 @@
+        self.camera = 'DBSPb'
+    
+    def compound_meta(self, headarr, meta_key):
+        retval = super(P200DBSPBlueSpectrograph, self).compound_meta(headarr, meta_key)
+        if retval is not None:
+            return retval
+        if meta_key == 'binning':
+            binspatial, binspec = headarr[0]['CCDSUM'].split(' ')
+            return parse.binning2string(binspec, binspatial)
+        else:
+            msgs.error("Not ready for this compound meta")
+
+    def get_detector_par(self, hdu, det):
+        """
+        Return a DectectorContainer for the current image
+
+        Args:
+            hdu (`astropy.io.fits.HDUList`):
+                HDUList of the image of interest.
+                Ought to be the raw file, or else..
+            det (int):
+
+        Returns:
+            :class:`pypeit.images.detector_container.DetectorContainer`:
+
+        """
+        # Binning
+        binning = self.get_meta_value(self.get_headarr(hdu), 'binning')  # Could this be detector dependent??
+
+        # Detector 1
+        detector_dict = dict(
+            binning         = binning,
+            det             = 1,
+            dataext         = 0,
+            specaxis        = 0,
+            specflip        = True,
+            spatflip        = False, # check
+            platescale      = 0.389,
+            darkcurr        = 0.0,
+            saturation      = 65000.,
+            nonlinear       = 62./65.,
+            mincounts       = -1e10, # cross-check
+            numamplifiers   = 1,
+            gain            = np.atleast_1d(0.72),
+            ronoise         = np.atleast_1d(2.5),
+            datasec         = np.atleast_1d('[1:2835,1:410]'),       # TODO: from datamodel, should just be able to point this to DSEC1/BSEC1
+            oscansec        = np.atleast_1d('[1:2835,411:460]')      # but that just doesn't work, so we will hardcode.
+            )
+        return detector_container.DetectorContainer(**detector_dict)
+
+
+    def default_pypeit_par(self):
+        """
+        Set default parameters for P200 DBSPb reductions.
+        """
+        par = pypeitpar.PypeItPar()
+        par['rdx']['spectrograph'] = 'p200_dbsp_blue'
+
+        # Ignore PCA
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+
+
+        # JFH Is this correct?
+        # Processing steps
+        # turn_off = dict(use_illumflat=False)
+        # par.reset_all_processimages_par(**turn_off)
+
+        # Turn off the overscan
+        #for ftype in par['calibrations'].keys():
+        #    try:
+        #        par['calibrations'][ftype]['process']['overscan'] = 'none'
+        #    except (TypeError, KeyError):
+        #        pass
+        par['scienceframe']['process']['use_overscan'] = True
+        # Make a bad pixel mask
+        par['calibrations']['bpm_usebias'] = True
+        # Set pixel flat combination method
+        par['calibrations']['master_dir'] = 'Masters_Blue'
```

Comment:
Is this so you can reduce blue and red in the same folder?
I worry there will be unanticipated consequences (e.g. the scripts
may assume `Masters` in a few places).  I suggest avoiding.